src/background/telemetry.ts

```diff
@@ -161,7 +130,7 @@ const throttledInit = throttle(_init, 30 * 60 * 1000, {
 export const initUID = liftBackground(
   "INIT_UID",
   async (): Promise<void> => {
-    if (!(await _getDNT())) {
+    if (!(await getDNT())) {
```

Comment:
A boy scout would also rename this to avoid double negations. Yay or nay?

```suggestion
    if (await canTrack()) {
```

Reply:
I totally agree - I get confused by it whenever I have to read this code